e2e/process-services-cloud/edit-task-filters-component.e2e.ts

```diff
@@ -110,7 +110,7 @@ describe('Edit task filters cloud', () => {
 
         await expect(await tasksCloudDemoPage.editTaskFilterCloudComponent().checkSaveButtonIsEnabled()).toEqual(false);
         await expect(await tasksCloudDemoPage.editTaskFilterCloudComponent().checkSaveAsButtonIsEnabled()).toEqual(false);
-        await expect(await tasksCloudDemoPage.editTaskFilterCloudComponent().checkDeleteButtonIsEnabled()).toEqual(true);
```

Comment:
it('[C306896] Delete Save and Save as actions should be displayed when clicking on custom filter header', async () => {
I think you need to rename also the e2e on testrail

Reply:
Updated